src/components/Quote.module.css

```diff
@@ -0,0 +1,55 @@
+.section {
+  height: 86vh;
+}
+
+.section article {
+  height: 80vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 70px;
+}
+
+.section article figure {
+  width: 300px;
+}
+
+.section article figure blockquote p {
+  font-size: 15px;
+}
```

Comment:
- [ ] [**OPTIONAL**]  To make your style looks more pretty, I'll suggest you add the margin-top of the `blockquote paragraph` so that you could avoid these two parts( first paragraph of your quotes and home navigation link) to be tangle as you can see on the picture below:
![image](https://user-images.githubusercontent.com/71874425/177432311-c637d6ce-77a0-4b85-bcef-89a78a5728d3.png)